docs/how-to-guides/propagating-updates.md

```diff
@@ -0,0 +1,64 @@
+# Propagating updates 
+
+This resource seeks to offer direction for those needing to provide updates to content, whether in a template for a theme, pattern, or a block over an entire site. 
```

Comment:
The introduction and title weren't clear to me until I read the rest of the document. I believe it's essential to have a more descriptive title or ensure that this page gets linked from other places in the documentation. Unless it's a page that would get linked from articles and tutorials outside of the Block Editor Handbook.

Reply:
I'm really struggling with better titles 😅 but here are some ideas:
- How to propagate updates to different configuration of blocks
- How to propagate updates to content made of blocks
- Approaches for propagating updates to blocks
- Propagating updates for blocks, template parts, etc

Reply:
I like the last one the most, maybe with slight rewording:

> Propagating updates for blocks

or

> Propagating updates for block types 

We technically expect to see changes applied to block types to be reflected on the site.

Reply:
Went with the latter and updated the doc + manifest.json with the new title. Thank you for talking this out.